test/dummy/db/migrate/20161103100644_create_log_other_archive.rb

```diff
@@ -0,0 +1,25 @@
+class CreateLogOtherArchive < ActiveRecord::Migration
+  def up
+    set_connection do
+      create_table :log_others_archive do |t|
+        t.references :post
+        t.string :note
+      end
+    end
+  end
+
+  def down
+    set_connection do
+      drop_table :log_others_archive
+    end
+  end
+
+  def set_connection
+    connection_was = @connection
+    @connection = ActiveRecord::Base.establish_connection(:"other_#{Rails.env}").connection
+    yield
+    @connection = connection_was
```

Comment:
Needs an `ensure` just above here:

```ruby
  yield
ensure
  @connection = connection_was
```

Alternatively, would just overriding the `connection` method work?

```ruby
def self.connection
  LogOthers::Archive.connection # Or `ActiveRecord::Base.establish_connection(:"other_#{Rails.env}").connection`
end
```